.notes/justin/worklogs/2025-10-01-programmatic-navigation-api.md

```diff
@@ -0,0 +1,22 @@
+# Programmatic Navigation API
```

Comment:
@justinvdm I am fairly happy to have this live here; but I don't know how you feel about it. Maybe we should consider unifying these?